rootex/core/renderer/materials/basic_material.h

```diff
@@ -4,17 +4,21 @@
 
 class Texture;
 
-class TexturedMaterial : public Material
+class BasicMaterial : public Material
 {
 	DiffuseShader* m_DiffuseShader;
 	Ref<Texture> m_DiffuseTexture;
 	Microsoft::WRL::ComPtr<ID3D11SamplerState> m_SamplerState;
 
 	ImageResourceFile* m_ImageFile;
 
+	bool m_IsLit = true;
```

Comment:
Is the default true value needed?

Reply:
I had done that in order to automatically port the JSON files, removing now.